src/server/src/main/java/io/cassandrareaper/management/jmx/JmxCassandraManagementProxy.java

```diff
@@ -617,14 +618,14 @@ private int triggerRepairPost2dot2(
     Map<String, String> options = new HashMap<>();
 
     options.put(RepairOption.PARALLELISM_KEY, repairParallelism.getName());
-    options.put(RepairOption.INCREMENTAL_KEY, Boolean.toString(!fullRepair));
+    options.put(RepairOption.INCREMENTAL_KEY, Boolean.toString(repairType != RepairType.SUBRANGE_FULL));
```

Comment:
nit: for readability maybe add methods to enum
`repairType.isFull()`
and
`repairType.isIncrement()`
(and `repairType.isSubrange()`)

Reply:
yep, good idea 👍